app/models/user.rb

```diff
@@ -1,6 +1,9 @@
+# frozen_string_literal: true
+
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+         :recoverable, :rememberable, :validatable,
+         :confirmable
```

Comment:
nope, we don't need that now. we should keep registration as simple as possible without any annoyence. we don't have real users (and bots as well). we should make sign up for user as smooth as possible.